cuqi/experimental/mcmc/_pcn.py

```diff
@@ -3,7 +3,7 @@
 from cuqi.experimental.mcmc import SamplerNew
 from cuqi.array import CUQIarray
 
-class pCNNew(SamplerNew):  # Refactor to Proposal-based sampler?
+class PCNNew(SamplerNew):  # Refactor to Proposal-based sampler?
 
     _STATE_KEYS = SamplerNew._STATE_KEYS.union({'scale', 'current_likelihood_logd'})
```

Comment:
Looks to me like "lambd" should be part of the STATE_KEYS

Reply:
I moved initialization of `lambd` and `star_acc` to constructor (as in `MHNew` in initializing `_scale_temp`), and added `lambd` to `STATE_KEYS`.

Reply:
I do not see lambd added to the state keys?

Reply:
Sorry : ( Just added.